react/features/conference/middleware.any.ts

```diff
@@ -191,8 +191,9 @@ function _checkIframe(state: IReduxState, dispatch: IStore['dispatch']) {
         }
     }
 
-    if (inIframe() && state['features/base/config'].disableIframeAPI && !browser.isElectron()
-        && !isVpaasMeeting(state) && !allowIframe) {
+    // TODO: enable for mobile too?
```

Comment:
We'd need to change the notification payload because it has a link.

Reply:
I'm also wondering why we don't do this for electron... 

This is not related to the PR or the changes here.

Reply:
Because there is no way to reliably detect that it's an Electron we approve of and not some random app.

Reply:
Oh. so do you mean that in order to allow our electron app to work on a deployment where config.disableIframeAPI=true we allow all electron clients?

Reply:
Pretty much yeah.

Reply:
Done.